PurchasesTests/Purchasing/StoreKitRequestFetcherTests.swift

```diff
@@ -94,41 +83,13 @@ class StoreKitRequestFetcher: XCTestCase {
 
     func testCallsStartOnRequest() {
         setupFetcher(fails: false)
-        expect((self.factory!.requests[0] as! MockProductsRequest).startCalled).toEventually(beTrue(), timeout: .seconds(1))
+        expect((self.factory!.requests[0] as! MockReceiptRequest).startCalled).toEventually(beTrue(), timeout: .seconds(1))
     }
-
-    func testReturnsProducts() {
```

Comment:
these tests are removed since the class no longer takes care of this, but we do have tests for [these things in the products manager](https://github.com/RevenueCat/purchases-ios/blob/main/PurchasesCoreSwiftTests/Purchasing/ProductsManagerTests.swift)